ReactSkia/components/RSkComponent.cpp

```diff
@@ -65,13 +66,13 @@ RnsShell::LayerInvalidateMask RSkComponent::updateProps(const ShadowView &newSha
    updateMask= updateComponentProps(newShadowView,forceUpdate);
   //opacity
    if((forceUpdate) || (oldviewProps.opacity != newviewProps.opacity)) {
-      layer_->opacity = newviewProps.opacity;
+      layer_->opacity = (newviewProps.opacity > 1.0)? 1.0*MAX_8BIT:newviewProps.opacity*MAX_8BIT;
```

Comment:
can this condition be optimized further as 
layer_->opacity = ((newviewProps.opacity > 1.0)? 1.0:newviewProps.opacity)*MAX_8BIT